nncf/tensorflow/nncf_network.py

```diff
@@ -0,0 +1,37 @@
+
+import tensorflow as tf
+
+
+class NNCFNetwork(tf.keras.Model):
+    def __init__(self, target_model):
+        super().__init__(target_model)
```

Comment:
According to [docs](https://www.tensorflow.org/api_docs/python/tf/keras/Model) tf.kearas.Model `__init__` has 3 parameters: `inputs`, `outputs` and `name`. It's not convenient for me what happened when target model is passed there